openvdb_houdini/houdini/SOP_OpenVDB_Advect.cc

```diff
@@ -295,11 +295,11 @@ newSopOperator(OP_OperatorTable* table)
         .setDefault(temporalIntegrationSchemeToString(TemporalIntegrationScheme(0))));
 
     // Register this operator.
-    hvdb::OpenVDBOpFactory("OpenVDB Advect",
+    hvdb::OpenVDBOpFactory("VDB Advect",
         SOP_OpenVDB_Advect::factory, parms, *table)
         .setObsoleteParms(obsoleteParms)
-        .addAlias("OpenVDB Advect Level Set")
```

Comment:
Just a thought: we could probably safely remove all of these aliases now.

Reply:
Done!